[enterprise-4.14] OSDOCS-7556: add RN entry (#64903)
openshift/openshift-docs: release_notes/ocp-4-14-release-notes.adoc

[id="ocp-4-14-machine-config-operator"]
=== Machine Config Operator

[id="ocp-4-14-additional-prometheus-metrics"]
==== Additional metrics available in Prometheus

> With this release, you can query additional metrics that let you more closely monitor the state of your machines and machine config pools.

Reviewer: `let you` would be slightly informal and sounds synonymous to `allow` that ISG does not recommend to be used.
A better wording I could think of. You could choose a better one too.

```suggestion
With this release, you can query additional metrics that help in detailed monitoring of the state of your machines and machine config pools.
```

Author: updated

> 

Reviewer: Since the  general term `additional metrics` is used, we could also provide a link to the documentation where these metrics are listed out.

Author: There isn't a doc that lists out the monitoring targets to that level of granularity, just a list of default targets. But there is a doc that details how to use Prometheus and view a list of available metrics, so maybe that could be linked here 🙂
